options/locale/locale_en-US.ini

```
@@ -1062,7 +1062,8 @@ pulls.cannot_auto_merge_helper = Merge manually to resolve the conflicts.
 pulls.no_merge_desc = This pull request cannot be merged because all repository merge options are disabled.
 pulls.no_merge_helper = Enable merge options in the repository settings or merge the pull request manually.
 pulls.no_merge_wip = This pull request can not be merged because it is marked as being a work in progress.
-pulls.no_merge_status_check = This pull request cannot be merged because not all required status checkes are successful.
+pulls.no_merge_not_ready = This pull request is not ready to be merged, check status of reviews and status checks.
```

```suggestion
pulls.no_merge_not_ready = This pull request is not ready to be merged, check status of reviews and checks.
```
A bit ineffective I think

---

I know it is a bit double of _status. I think _status checks_ is rather established term so good to keep that, but I reformulated a bit.